### PR TITLE
Rename YAML file

### PR DESCRIPTION
**Pull request by alexpapworth**
We added customer-api-docs recently, so got our Rails app to output two different YAML files, depending on which Docs repository it's going to.

Also added in tags to group the requests into different categories.

### CODE REVIEW

File: resellers_api.yaml [hunk 1]
```diff
@@ -1,83 +1,280 @@
...
+                name: Carehome
+                type_of_organisation: Social Care
+                timezone: London
+                reseller_organisation_id: '101'
+                created_at: '2024-02-02T15:40:14Z'
+                locations:
+                - name: Alexander Fleming House
+                  timezone: London
+                  regulator_reference: CQC45678
+                  postcode:
```
Comment (alexpapworth) [inline resellers_api.yaml]: Not sure why these postcodes are coming up as blank now. Do the tests not work anymore?

Comment (SimonMerrikin) [reply resellers_api.yaml]: `/requests/resellers_spec.rb` doesn't have seeded postcodes. `/controllers/resellers_controller_spec.rb` does, but I see now that that file isn't the one read for building the yaml?

So we need to make sure seeded data we want to see is in `spec/requests/resellers_spec.rb`

Comment (alexpapworth) [reply resellers_api.yaml]: Ah I see. I'll update the request specs :+1:

**Comment by SimonMerrikin** [comment]
I assume we should merge this. Working off this branch for PR https://github.com/digital-reception/reseller-api-docs/pull/10

**Comment by alexpapworth** [comment]
Ah yep, good point. Merging now.

**Pull request merged**